app/etl/ga/user_feedback_processor.rb

```diff
@@ -0,0 +1,87 @@
+class GA::UserFeedbackProcessor
+  include Concerns::Traceable
+
+  def self.process(*args)
+    new(*args).process
+  end
+
+  def initialize(date:)
+    @date = date
+  end
+
+  def process
+    time(process: :ga) do
+      extract_events
+      transform_events
+      load_metrics
+    end
+  end
+
+private
+
+  def extract_events
+    batch = 1
+    ga_service.find_user_feedback_in_batches(date: date) do |events|
+      log process: :ga, message: "Processing #{events.length} events in batch #{batch}"
+      Events::GA.import(events, batch_size: 10_000)
+      batch += 1
+    end
+  end
+
+  def transform_events
+    fix_invalid_prefix_in_page_paths
+  end
+
+  def load_metrics
+    conn = ActiveRecord::Base.connection
+    date_to_s = date.strftime("%F")
+    conn.execute(load_metrics_query(date_to_s))
+    conn.execute(clean_up_query)
+  end
+
+  def load_metrics_query(date_to_s)
+    <<~SQL
+      UPDATE facts_metrics
+      SET is_this_useful_no = s.is_this_useful_no,
+          is_this_useful_yes = s.is_this_useful_yes
+      FROM (
+        SELECT is_this_useful_no,
+               is_this_useful_yes,
+               dimensions_items.id
+        FROM events_gas, dimensions_items
+        WHERE page_path = base_path
+              AND events_gas.date = '#{date_to_s}'
+      ) AS s
+      WHERE dimensions_item_id = s.id AND dimensions_date_id = '#{date_to_s}'
+    SQL
+  end
+
+  def clean_up_query
```

Comment:
I would only clean up events that has been created on my process (having `is_this_useful_no` or `is_this_useful_yes` with a value != to 0) It would need a test as well.